fidesctl/src/fidesctl/cli/commands/generate.py

```diff
@@ -65,7 +65,7 @@ def generate_dataset_okta(
     It will need to be run again if the tracked resources change.
     """
     try:
-        import okta
+        import okta  # pylint: disable=unused-import
```

Comment:
I think it would be more intuitive if these `try`/`catch` blocks were moved to just above the code where they're needed, instead of randomly importing packages here.

The same goes for the other 3 instances.

Reply:
thanks @PSalant726 ! 

These pylint errors were masked previously (due to the lack of `__init__.py` I am guessing), but I see what you mean. 

Will take a look to see if I notice any issue with moving them, I think there was a hangup with the difficulty in handling an optional dependency iirc but I should be able to validate if moving them is a problem for some reason. If I end up being a little less than confident, we can open up another issue for 1.6.0 if that works 👍🏽

Reply:
@PSalant726 let me know if this is what you had in mind -> 6294f15650e1a080d2774ffd3d6cf02a5393f2c3

Did a bit of testing and think it follows your suggestion and accomplishes the same workflow, with an added DRY bonus 🙌🏽

Reply:
looks good to me, expediency is important here as 1.5.0 is broken for literally everyone until this is pushed